packages/system/monitoring/templates/grafana/oidc-keycloak.yaml

```
@@ -0,0 +1,119 @@
+  operator CRDs would not be present and the chart would silently
+  no-op without the fail-fast.
+
+  The three KeycloakRealmGroup objects are chart-owned but their
+  membership is not — a Keycloak operator adds users to them via the
+  Keycloak UI or via a KeycloakRealmUser CR. Grafana's
+  `role_attribute_path` (see grafana.yaml) evaluates group membership
+  on login and assigns Admin / Editor / Viewer accordingly.
+*/}}
+{{- if eq .Values.oidc.mode "System" }}
+{{-   include "monitoring.oidc.assertSystemEnabled" . }}
+{{-   if .Capabilities.APIVersions.Has "v1.edp.epam.com/v1" }}
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)